.github/workflows/ci-cd.yml

```
@@ -47,8 +47,9 @@ jobs:
           --health-timeout=5s
           --health-retries=5
     env:
-      DATABASE_URL: postgres://local_user:local_password@localhost:5432/local_db
-      PORT: 3001
+        DATABASE_URL: ${{ secrets.DATABASE_URL }}
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 8

(indentation)